vivisect/parsers/elf.py

```diff
@@ -597,7 +597,7 @@ def applyRelocs(elf, vw, addbase=False, baseaddr=0):
                         vw.makeImport(rlva, "*", dmglname)
                         vw.setComment(rlva, name)
 
-                    elif rtype in (Elf.R_386_32, Elf.R_386_COPY):
+                    elif rtype in (Elf.R_386_32, Elf.R_386_COPY, Elf.R_X86_64_TPOFF64):
```

Comment:
Soo, since this change prompted 0 complaints in our unittests, can you give a rundown of what this changes/fixes?

Reply:
this is just telling the ELF parser that this type of relocation is normal, and not to complain about "not handling it".